tests/PageTypes/DatedUpdateHolderControllerTest.php

```diff
@@ -66,6 +67,7 @@ public function testInvalidDateFormat()
      */
     private function isRunningMySQL()
     {
-        return strpos(strtolower(get_class(DB::get_connector())), 'mysql') !== false;
+        $connector = DB::get_connector();
+        return strpos(strtolower(get_class($connector)), 'mysql') !== false || $connector instanceof PDOConnector;
```

Comment:
I think postgres can also run PDO? (we don't test this in our matrix though)

Reply:
Hmmm that's a good point... since we don't have postgres PDO in our matrix I think I'd prefer to keep this change - the likelyhood of someone running this test with postgres PDO is very unlikely (and they can find a better way to do this if they want to) - but for now we _do_ have mysql PDO tests.

Reply:
so how come strpos(strtolower(get_class(DB::get_connector())), 'mysql') isn't working? What does DB::get_connector() return with mysql pdo?

Reply:
The PDO connector class is literally `PDOConnector` (the full namespaced class name is checked of course, so `SilverStripe\ORM\Connect\PDOConnector`)
So it's a mysql db, but the connector class name doesn't hint at that at all.